server/documents/elements/header.html.eco

```diff
@@ -256,6 +256,35 @@ themes      : ['Default', 'Classic', 'Bookish', 'Chubby', 'Material']
     </h3>
   </div>
 
+  <div class="example" data-since="2.9.4">
+    <h4 class="ui header">Seamless attached</h4>
+    <p>A top header can be seamless attached without related attached border.</p>
+    <h3 class="ui seamless top attached header">
+      Seamless Top Attached
+    </h3>
+    <div class="ui attached segment">
+      <img class="ui wireframe image" src="/images/wireframe/paragraph.png">
+    </div>
+    <div class="ui attached segment">
+      <img class="ui wireframe image" src="/images/wireframe/paragraph.png">
+    </div>
+  </div>
+
+  <div class="example" data-since="2.9.4">
```

Comment:
`data-since` has no effect here as no title is given. Therefore it is `another example`  and `data-class` is again needed for syntax highlighting
```suggestion
  <div class="another example" data-class="seamless,attached">
```